apps/chat/src/backend/serverless/template.yaml

```diff
@@ -35,138 +35,250 @@ Resources:
+
+
+  # Trigger Lambda function to create Amazon Chime App Instance and channel flow creation
   TriggerChimeAppInstanceLambda:
-      Type: AWS::CloudFormation::CustomResource
-      Properties: 
-        ServiceToken: !GetAtt ChimeAppInstanceLambda.Arn
+    Type: AWS::CloudFormation::CustomResource
+    DependsOn: ProfanityAndDLPProcessor
+    Properties:
+      ServiceToken: !GetAtt ChimeAppInstanceLambda.Arn
 
   #Lambda that creates AWS Chime App admin user
   ChimeAppAdminLambda:
-      DependsOn: TriggerChimeAppInstanceLambda
-      Type: "AWS::Lambda::Function"
-      Properties:
-        Handler: "index.handler"
-        Role: !GetAtt LambdaExecuteRole.Arn
-        Runtime: "nodejs12.x"
-        Timeout: 60
-        Layers: 
-          - !Ref AWSSDKChimeLayer
-        Code:
-          ZipFile: >
-            "use strict";
-            const AWS = require("aws-sdk");
-            const uuidv4 = require("uuid");
-            const response = require("cfn-response"); 
-            AWS.config.update({ region: process.env.AWS_REGION });
-            const chime = new AWS.Chime({ region: process.env.AWS_REGION }); 
-
-            const { CHIME_APP_INSTANCE_ARN } = process.env;
-            
-            exports.handler = async (event, context, callback) => {
-              console.log("Event: \n", event);
-              console.log("Create Chime SDK App Admin");
-              if (event["RequestType"] === "Create") {
-                //create a chime app user
-                const createUserParams = {
-                  AppInstanceArn: CHIME_APP_INSTANCE_ARN,
-                  AppInstanceUserId: `Admin`,
-                  ClientRequestToken: `${uuidv4()}`,
-                  Name: `Admin`
-                };
-                
-                try {
-                  const userResponse = await chime.createAppInstanceUser(createUserParams).promise();
-
-                  console.log("Successfully created user");
-                  console.log(userResponse); // successful response
-                } catch (error) {
-                  console.log("ERROR CAUGHT \n", error);
-                }
-
-                //create a chime app admin
-                const createAdminParams = {
-                  AppInstanceArn: CHIME_APP_INSTANCE_ARN,
-                  AppInstanceAdminArn: CHIME_APP_INSTANCE_ARN + '/user/Admin'
-                };
+    DependsOn: TriggerChimeAppInstanceLambda
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Handler: "index.handler"
+      Role: !GetAtt LambdaExecuteRole.Arn
+      Runtime: "nodejs12.x"
+      Timeout: 60
+      Layers:
+        - !Ref AWSSDKChimeLayer
+      Code:
+        ZipFile: >
+          "use strict";
+          const AWS = require("aws-sdk");
+          const uuidv4 = require("uuid");
+          const response = require("cfn-response");
+          AWS.config.update({ region: process.env.AWS_REGION });
+          const chime = new AWS.Chime({ region: process.env.AWS_REGION });

```

Comment:
Will you be updating this to use the new Identity and Messaging SDKs?

Reply:
updated